{"name": "Создание дублирующей сущности курьера. Ожидаемый результат: 409", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\n  - Этот логин уже используется\n  + Этот логин уже используется. Попробуйте другой.", "trace": "self = <test_create_courier.TestCreateCourier object at 0x0000022D04F76410>, create_new_user_and_return_login_password = <Response [201]>\n\n    @allure.title('Создание дублирующей сущности курьера. Ожидаемый результат: 409')\n    def test_create_courier_already_created_conflict_409(self, create_new_user_and_return_login_password):\n        # Arrange\n        response_by_create_courier = create_new_user_and_return_login_password\n        data = json.loads(response_by_create_courier.request.body)\n        # Act\n        repeated_response = ApiClient.post(url=urls.COURIER_URL, data=data)\n        # Assert\n        assert repeated_response.status_code == 409\n>       assert repeated_response.json()['message'] == CourierData.login_taken_response\nE       AssertionError: assert 'Этот логин у...буйте другой.' == 'Этот логин уже используется'\nE         - Этот логин уже используется\nE         + Этот логин уже используется. Попробуйте другой.\n\ntests\\test_create_courier.py:32: AssertionError"}, "steps": [{"name": "Вызываем метод POST c данными - {'login': 'iOPj11O5cv', 'password': 'bZWGk9HZ8d', 'firstName': 'E9TVouKq1F'}, эндпоинт - 'https://qa-scooter.praktikum-services.ru/api/v1/courier'", "status": "passed", "parameters": [{"name": "url", "value": "'https://qa-scooter.praktikum-services.ru/api/v1/courier'"}, {"name": "data", "value": "{'login': 'iOPj11O5cv', 'password': 'bZWGk9HZ8d', 'firstName': 'E9TVouKq1F'}"}], "start": 1741703247404, "stop": 1741703247833}], "start": 1741703247404, "stop": 1741703247835, "uuid": "f237e7f8-596e-477e-8f83-8dd8806c9210", "historyId": "045b2dd30fc2dfc7bf4709fdd52d94ec", "testCaseId": "045b2dd30fc2dfc7bf4709fdd52d94ec", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_courier_already_created_conflict_409", "labels": [{"name": "feature", "value": "Сценарии создания курьера"}, {"name": "tag", "value": "create_courier"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "WS015"}, {"name": "thread", "value": "8152-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}